src/scss/_date-and-venue.scss

```diff
@@ -0,0 +1,95 @@
+.date {
+  color: $text-color;
+  padding: 60px 0;
+  text-align: center;
+
+  .row {
+    display: flex;
+    justify-content: space-around;
+    margin: 0 auto;
+    max-width: 640px;
+  }
+
+  .column {
+    align-content: space-between;
+    display: flex;
+    flex-direction: column;
+    font-weight: 500;

```

Comment:
Does font contain such weight, or we should use here 400 or 700?

Reply:
It's quite strange, it seems like there's 400, 600, 700, but when I apply 500 as a font-weight it works - the weight is somwhere between 400 and 600. I'll investigate that alongside solving issue #33.